scripts/index_d_ts

```diff
@@ -483,13 +485,15 @@ declare module "SendbirdUIKitGlobal" {
     onCancel: () => void;
   };
 
-  export enum VoicePlayerStatus {
+  export enum VoicePlayerStatusType {
     IDLE = 'IDLE',
     PREPARING = 'PREPARING',
     READY_TO_PLAY = 'READY_TO_PLAY',
     PLAYING = 'PLAYING',
     COMPLETED = 'COMPLETED',
   }
+  // should be deprecated / for backward
+  export type VoicePlayerStatus = VoicePlayerStatusType;
```

Comment:
👍 better comment in `src/hooks/VoicePlayer/dux/initialState.ts` ; we want to generate the index.d.ts file in the future :)